.travis.yml

```diff
@@ -16,11 +16,16 @@ python:
 node_js:
 - '0.10'
 addons:
+  artifacts:
+    paths:
+    - results
+  firefox: "latest-esr"
```

Comment:
Slimer is erroring out and all I can guess is this is causing it. I thought they included the latest ESR by default though so this might not be it. What happens if you remove this line?